responsibleai/tests/common_utils.py

```diff
@@ -189,8 +191,23 @@ def create_housing_data(create_small_dataset=True):
     return x_train, x_test, y_train, y_test, housing.feature_names
 
 
+class FetchDiceAdultCensusIncomeDataset(object):
+    def __init__(self):
+        pass
+
+    def fetch(self):
+        return helpers.load_adult_income_dataset()
+
+
 def create_adult_income_dataset(create_small_dataset=True):
-    dataset = helpers.load_adult_income_dataset()
+    fetcher = FetchDiceAdultCensusIncomeDataset()
+    action_name = "Dataset download"
```

Comment:
Can we change the action name to "Adult dataset download"?

Reply:
done!